mcodingbot/plugins/bot_management.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+import crescent
+import hikari
+
+from mcodingbot.config import CONFIG
+
+plugin = crescent.Plugin()
+
+
+@plugin.include
+@crescent.command(
+    name="restart",
+    description="Restarts the bot.",
+    default_member_permissions=hikari.Permissions.MODERATE_MEMBERS,
```

Comment:
I feel like this one makes more sense tbh 
```suggestion
    default_member_permissions=hikari.Permissions.ADMINISTRATOR,
```